lti_consumer/models.py

```diff
@@ -316,10 +354,33 @@ def _get_lti_1p3_consumer(self):
                 consumer.enable_nrps(get_lti_nrps_context_membership_url(self.id))
 
             return consumer
+        elif self.config_store == self.CONFIG_ON_DB:
+            # TODO: I couldn't quite tell whether the Proctoring Services Specification relies on LTI Advantage or
+            #       just LTI 1.3, but I think, according to this document, it's a proposed LTI Advantage service,
+            #       so I'm using the LtiAdvantageConsumer.
+            #       https://www.imsglobal.org/activity/learning-tools-interoperability
```

Comment:
@MichaelRoytman All LTI Advantage services only rely on the base LTI 1.3 spec and are completely independent between themselves. 
Come to think of it, it might be best to keep the proctoring service inside `LtiAdvantageConsumer` to avoid and extra fork here when configuring the class, what do you say?

Reply:
That makes sense. We either need a consumer class for each LTI service (or, at least an extra one for proctoring if we keep the LtiAdvantageConsumer as one) and then have to decide which consumer class to use depending on whether we're using proctoring or not, or we need to have all services implemented in `LtiAdvantageConsumer`, which allows us to avoid the forking but makes it bloated.

On the other hand, once the library is refactored to be able to be used outside an xBlock, users of the library may not want all LTI services, right? edx-exams will not want Assignment and Grade Services, for example. I think having separate classes might allow us to gate what features are supported for an installation.